src/assets/i18n/sv.json

```diff
@@ -25,8 +25,8 @@
   ,"About ZoneMaster Text":"Ge din domän en komplett hälsoundersökning! Zonemaster hjälper dig att kontrollera hur din webbplats mår. På köpet får du dessutom en bättre förståelse för hur domämnamnssystemet DNS fungerar.\nNär en domän (även kallad zon) skickas till Zonemaster, undersöker programmet domänens hälsotillstånd från början till slut. Det görs genom att Zonemaster går igenom DNS från roten (.) till TLD:n (toppdomänen, till exempel .se) och slutligen de DNS-servrar som innehåller information om den specificerade domänen (till exempel iis.se).\nZonemaster passar även på att utföra en hel del andra tester, exempelvis kontroll av DNSSEC-signaturer, att olika värdar går att komma åt och att IP-adresser är giltiga. Allt för att kontrollera att din domän mår så bra som möjligt."
   ,"About domain name system, DNS": "Om domain name system, DNS"
   ,"About DNS Text":"De flesta som använder internet har aldrig ens reflekterat över vad som egentligen händer när man har skrivit in en adress i sin webbläsare och tryckt ned enter-tangenten. Något förenklat är domännamnssystemet (DNS) en databas som översätter domännamn till IP-adresser, på samma sätt som en telefonkatalog översätter namn till telefonnummer. En IP-adress är en unik nummerserie som identifierar varje dator som är exponerad mot internet. Det kan liknas vid hur varje telefon har ett eget telefonnummer i telenätet. Långa nummerserier fungerar fint för datorer, men för oss människor är de svåra att memorera. Därför har man utvecklat DNS ett extra system för oss när vi vill nå fram till en webbserver med en viss sajt på eller en e-postserver som vi vill skicka e-postmeddelanden till."
-  ,"No data for this test": "No data for this test"
-  ,"The domain name contains unauthorized character(s)": "The domain name contains unauthorized character(s)"
+  ,"No data for this test": "Ingen data för detta test"
+  ,"The domain name contains unauthorized character(s)": "Domännamnet innehåller otillåtet tecken"
```

Comment:
Weird grammar. I'd prefer "minst ett otillåtet".

Reply:
But the string is probably never used anyway.

Reply:
Should we keep it if it's not used?

Reply:
No, and that this what I've asked @pamasse to check. If not used, it should be removed from all the json files.